.trivyignore

```diff
@@ -1,17 +1,4 @@
-# telegraf MEDIUM
-CVE-2019-3826
-CVE-2023-28841
-CVE-2023-28842
-CVE-2023-25809
-GHSA-2w8w-qhg4-f78j
-
-# telegraf HIGH
-CVE-2023-27561
-CVE-2023-28840
-CVE-2023-28642
-CVE-2023-2253
-CVE-2023-34231
-
+#telegraf MEDIUM
 GHSA-2w8w-qhg4-f78j
```

Comment:
is this not addressed with telegraf upgrade?

Reply:
No this will be in the next version 1.27.3